examples/logchimp/README.md

```diff
@@ -0,0 +1,14 @@
+---
+title: LogChimp
+description: Track your customers feedback to build better products with LogChimp
+buttonSource: https://github.com/logchimp/deploy-buttons
+tags:
```

Comment:
Add `typescript` as well?

Reply:
LogChimp codebase is not written in TS, yet! 😅

Reply:
Added `javascript` tag instead.